wagtail/admin/templates/wagtailadmin/shared/dialog/dialog.html

```diff
@@ -29,14 +29,17 @@
             {% endif %}
 
             <div class="w-dialog__content">
-                <h2 class="w-dialog__title w-h1" id="title-{{ id }}">
-                    {% if icon_name %}{% icon name=icon_name classname="w-dialog__icon" %}{% endif %}
-                    {{ title }}
-                </h2>
-
-                {% if subtitle %}
-                    <p class="w-dialog__subtitle w-help-text">{{ subtitle }}</p>
-                {% endif %}
+                <div class="w-dialog__header">
+                    <h2 class="w-dialog__title w-h1" id="title-{{ id }}">
+                        {% if icon_name %}{% icon name=icon_name classname="w-dialog__icon" %}{% endif %}
+                        {{ title }}
+                    </h2>
+                    {% if subtitle %}
+                        <p class="w-dialog__subtitle w-help-text">{{ subtitle }}
+                            {% if a11y_count %}<span class="w-a11y-result__count" data-a11y-result-count></span>{% endif %}
```

Comment:
This dialog component is used for other scenarios than the checker, so this doesn’t seem appropriate to me (the dialog component is meant to be generic). We could however use a fragment to insert this in the subtitle:


```jinja
{% fragment as issues_found %}
{% trans 'Issues found' %} {% if a11y_count %}<span class="w-a11y-result__count" data-a11y-result-count></span>{% endif %}
{% endfragment %}
```